src/corner_kick/config/jest.config.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * The configuration file for Jest
+ *
+ * Jest is the library used for unit testing components
+ * such as actions and reducers
+ */
+module.exports = {
+    rootDir: '..',
+    roots: ['<rootDir>/src'],
+    // Assigning all tsx files to be interpreted by ts-jest
+    transform: {
+        '^.+\\.tsx?$': 'ts-jest',
+    },
+    // Any files inside __tests__ is a test file
```

Comment:
Why `__tests__` instead of `tests`? Do the underscores have some significance?

Reply:
Allows folder to be at the top of the parent directory. It's a convention with Jest

Reply:
Also Jest looks specifically for this type of folder name for tests